docker/LICENSE

```diff
@@ -0,0 +1,202 @@
+Apache License
```

Comment:
Why did we add a license file here?  We already have one at the root of the repo.